P/pugixml/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "pugixml"
+version = v"1.11.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://github.com/zeux/pugixml/releases/download/v1.11/pugixml-1.11.tar.gz", "26913d3e63b9c07431401cf826df17ed832a20d19333d043991e611d23beaa2c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -S pugixml-1.11 -B build
+cd build/
+make
+make install
+cd ..
+install_license pugixml-1.11/LICENSE.md
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("i686", "windows"; ),
+    Platform("x86_64", "windows"; )
+]
```

Comment:
Can we just build for all platforms and see what happens?
```suggestion
platforms = supported_platforms(; experimental=true)
```